.github/workflows/python-ci.yml

```
@@ -39,7 +39,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        python: ['3.8', '3.9', '3.10', '3.11']
+        python: ['3.8', '3.9', '3.10', '3.11', '3.12'']
```

This might be the reason:
```suggestion
        python: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

---

Thanks. Update.